src/cmd/webhook/builder.go

```diff
@@ -90,6 +91,24 @@ func addFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&certificateKeyFileName, FlagCertificateKeyFileName, "tls.key", "File name for the private key.")
 }
 
+func startCertificateWatcher(podName string, webhookManager manager.Manager, namespace string) error {
```

Comment:
```suggestion
func startCertificateWatcher(webhookManager manager.Manager, namespace string, podName string) error {
```

https://github.com/Dynatrace/dynatrace-operator/blob/main/doc/coding-style-guide.md#function-parameter-and-return-value-order